hardware/raspberrypi/bootmodes/msd.md

```diff
@@ -57,7 +57,9 @@ After five to ten seconds, the Raspberry Pi should begin booting and show the ra
 <a name="pi4"></a>
 ## Raspberry Pi 4
 
-The Raspberry Pi 4's bootcode is stored in [EEPROM](../booteeprom.md) and can be updated. Support for mass storage boot will be added in a future update.
+USB mass storage support is currently undergoing beta testing, once beta test is compelte the release version will enable USB mass storage boot by default. The `program_usb_boot_mode` is not required on Pi 4.
```

Comment:
Missing word - 'option' after `program_usb_boot_mode`

Reply:
> If the issues above have been resolved (can people please flag as resolved when they have fixed things), is this ready to go?

The issues I previously flagged have been resolved - there isn't a "resolved" button for some reason.

I've flagged 2 other minor issues though.